packages/lodestar/src/logger/winston.ts

```diff
@@ -17,9 +18,9 @@ export class WinstonLogger implements ILogger {
       ...options,
     };
     this.winston = createLogger({
-      level: LogLevel[LogLevel.debug], // log level switching handled in `createLogEntry`
+      level: LogLevel[LogLevel.verbose], // log level switching handled in `createLogEntry`
```

Comment:
enum gets converted to something like this at runtime:
```
{
  1: "debug",
  2: "verbose:,
  "debug: 1,
  "verbose: 2,
}
```
So the above is a roundabout way of doing:
```
  level: "verbose",
```
I think it shouldn't actually matter what `level` gets set to here, since according to the comment, the level gets written in `createLogEntry`